Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -468,7 +471,9 @@ void WebProcess::initializeWebProcess(WebProcessCreationParameters&& parameters,
 #if ENABLE(PERIODIC_MEMORY_MONITOR)
         if (auto pollInterval = parameters.memoryFootprintPollIntervalForTesting)
             memoryPressureHandler.setMemoryFootprintPollIntervalForTesting(pollInterval);
-        memoryPressureHandler.setShouldUsePeriodicMemoryMonitor(true);
+        // If we're running with FastMalloc disabled, some kind of testing or debugging is probably happening.
+        // Let's be nice and not enable the memory kill mechanism.
```

Comment:
This is probably only true if `USE(SYSTEM_MALLOC)` is false. If system malloc is enabled at build time then you're just detecting that bmalloc isn't supported.

Reply:
I copied and pasted this from the file below, but actually that makes a lot of sense, and might explain memory issues with MotionMark on ArmV7